nixos/tests/livebook-service.nix

```
@@ -0,0 +1,37 @@
+import ./make-test-python.nix ({ lib, pkgs, ...} : {
+  name = "livebook-service";
+
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      users.users.alice = {
+        isNormalUser = true;
+        uid = 1000;
+      };
```

There is this neat file that you can import to make the alice user.

```suggestion
      imports = [ ./common/user-account.nix ];
```

---

Neat.  Done.